### PR TITLE
feat(components/google-cloud):Add support for URI prefix

### PR DESCRIPTION
**Pull request by SinaChavoshi**
Changing URI to match the following prefixes: 

- `aiplatfrom://` - corresponding to AIPlatform rescue names.
- `bq://` - corresponding to BigQuery resource URI
- `gcs://` - corresponding to Google Cloud Storage resource URI

### CODE REVIEW

File: components/google-cloud/google_cloud_pipeline_components/aiplatform/utils.py [hunk 1]
```diff
@@ -547,7 +549,7 @@ def component_yaml_generator(**kwargs):
                 )
                 input_args.append(f'--{prefix_key}.{component_param_name}')
                 if is_mb_sdk_resource_noun_type(param_type):
-                    input_args.append(InputPathPlaceholder(input_name=key))
+                    input_args.append(InputUriPlaceholder(input_name=key))
```
Comment (sasha-gitg) [inline components/google-cloud/google_cloud_pipeline_components/aiplatform/utils.py:552]: How does this change the input?

**Review by sasha-gitg — APPROVED** [review]
/lgtm

File: components/google-cloud/google_cloud_pipeline_components/aiplatform/utils.py (hunk 1 above)
Comment (SinaChavoshi) [reply components/google-cloud/google_cloud_pipeline_components/aiplatform/utils.py:552]: sorry merged before i responded, using `InputPathPlaceholder` adds a `/gcs/` at the beginning of the input. so if the input is `http://aiplatform/v1/xyz` it becomes `/gcs/http://aiplatform/v1/xyz`. Using `InputUriPlaceholder` directly copies the URI without any modification. The code handles either case ( by attempting to remove gcs fuse prefix first ) however changing the input type makes it cleaner.

**Pull request merged**